### ISSUE TITLE
templates not working on actions

### ISSUE THREAD

**Issue by ocangelo** [Nerwyn/android-tv-card#108]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
3.8.1

**Last working release (if known):**

**Browser and Operating System:**
Windows - Chrome
Android - Native App

**Description of problem:**
Templates seem to be working fine when changing styles etc. but they don't work when defining actions.
in the example below "spotify" works fine, but both "plex" and "youtube" generate a "none" action.

**Javascript errors shown in the web inspector (if applicable):**
when clicking the button the value of "d.action" is "none":

```
                    if (d && this.handleConfirmation(d))
                        try {
                            switch (d.action) {
```

**Your Full Remote Config**

```
type: custom:android-tv-card
rows:
  - - plex
    - youtube
    - spotify
custom_actions:
  myselectsource:
    tap_action:
      action: call-service
      service: media_player.select_source
      target:
        entity_id: media_player.roku_ultra
  myselectsource2:
    service: media_player.select_source
    service_data:
      entity_id: media_player.roku_ultra
  plex:
    style:
      color: orange
    template: myselectsource
    tap_action:
      data:
        source: Plex - Free Movies & TV
  youtube:
    style:
      color: red
    template: myselectsource2
    service_data:
      source: YouTube
  spotify:
    style:
      color: green
    service: media_player.select_source
    service_data:
      entity_id: media_player.roku_ultra
      source: Spotify Music
```

**Additional information:**

**Comment by ocangelo** [comment]
that worked, thank you for the quick response!

**Comment by Nerwyn** [comment]
Fixed in [3.8.2](https://github.com/Nerwyn/android-tv-card/releases/tag/3.8.2).
